### DISCUSSION TITLE
Vitedge

### DISCUSSION THREAD

**Discussion by flikteoh** [ElMassimo/iles#58] [Ideas]
Just thought of sharing this: [Vitedge](https://vitedge.js.org/getting-started.html)

Looks very exciting. Wondering if it would be easy to setup with iles?

**Comment by ElMassimo** [comment]
Hi! I'm not familiar with the internals of Vitedge.

Depending on how it works, it might be possible to create an integration that builds an îles site targeting Vitedge, so it can be deployed in edge services like Cloudflare.

This would be particularly nice for sites with many [dynamic routes](https://iles-docs.netlify.app/guide/routing#dynamic-routes), or sites that need islands but have dynamic content.

**Comment by flikteoh** [comment]
Yeah it's quite interesting at the moment. iles is pretty good now anyway! This will mostly be good to have.

I'm still trying out before using iles on production. Mainly we intend to host on cloudflare for the most part and want to consider using the CF image resizing instead to reduce build time.

**Reply by ElMassimo** [reply]
Nice! While I've used Cloudflare Pages, I find the environment setup takes a long time. Netlify is consistently faster in my builds.

**Reply by flikteoh** [reply]
Yeah thats true. Their build pipeline is still being worked on. Currently I'm still using github for build, s3 for hosting and cf workers for caching and serving 😅
